webapp/package.json

```diff
@@ -20,6 +20,7 @@
     "intl": "1.2.5",
     "jasny-bootstrap": "3.1.3",
     "jquery": "3.2.1",
+    "key-mirror": "1.0.1",
```

Comment:
Also, why are we switching the key mirror library that we use? Was there an issue with the existing one?